### PR TITLE
Change Bug report issue template to Github Forms

### PR DESCRIPTION
**Pull request by st-Wook**
This PR migrates the bug report template to Github Forms!

Let me know if you have any advice! or If you don't want to, I'll close it!

<details><summary>Preview of new form</summary>

image

</details>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/--bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: GDevelop version
+      description: Which version of GDevelop are you used?
```
Comment (ClementPasteau) [inline .github/ISSUE_TEMPLATE/--bug-report.yml:28]: typo: `Which version of GDevelop are you using?`

File: .github/ISSUE_TEMPLATE/--bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        * Please include a link to a game if possible!
+        * If applicable, add screenshots to help explain your problem.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error
+    validations:
+      required: true
+  - type: dropdown
```
Comment (ClementPasteau) [inline .github/ISSUE_TEMPLATE/--bug-report.yml:25]: I would split this question in 2!
One for the web/desktop version, and one for the actual GDevelop version.
The first one can be a dropdown with web/desktop like you did.
The second one has to be a free text field, with a placeholder like: `5.1.159? 5.1.160?`

File: .github/ISSUE_TEMPLATE/--bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      required: true
+  - type: textarea
+    attributes:
+      label: Info
+      value: |
+        OS: 
+        OS Version: 
+        Browser(For Web app): 
+        Device(For Smartphone): 
+  - type: textarea
```
Comment (ClementPasteau) [inline .github/ISSUE_TEMPLATE/--bug-report.yml:43]: can we add screenshots in a textarea field?
If not, maybe we should add a new field especially for screenshots

**Review by ClementPasteau — COMMENTED** [review]
Thanks for updating this, I think it's a great addition!
A few comments to ensure we're not losing information from the existing template

File: .github/ISSUE_TEMPLATE/--bug-report.yml (hunk 2 above)
Comment (st-Wook) [reply .github/ISSUE_TEMPLATE/--bug-report.yml:25]: It seems that two types cannot be used with the same label.:cry:
So I divided it with the image below. And I changed the selection list item to [Desktop, Web]. What do you think?
image
image

File: .github/ISSUE_TEMPLATE/--bug-report.yml (hunk 3 above)
Comment (st-Wook) [reply .github/ISSUE_TEMPLATE/--bug-report.yml:43]: We can add screenshots or files to the textarea field! (same to comment textarea field)

Comment (st-Wook) [reply .github/ISSUE_TEMPLATE/--bug-report.yml:43]: image

**Review by ClementPasteau — COMMENTED** [review]
one last comment and good to go

File: .github/ISSUE_TEMPLATE/--bug-report.yml (hunk 2 above)
Comment (st-Wook) [reply .github/ISSUE_TEMPLATE/--bug-report.yml:25]: Done!

**Review by ClementPasteau — APPROVED** [review]
Thanks!

**Pull request merged**